tests/test_tifs.py

```diff
@@ -14,13 +14,18 @@
 _API = {'main': API_URL, 'develop': API_TEST_URL}
 
 
-def _get_tif_tolerances(file_name):
-    tif_type = file_name.name.split('_')[-1]
-    if tif_type == 'phase.tif':  # InSAR
-        return 0.0, 0.0
-    if tif_type == 'area.tif':  # RTC
+def _get_tif_tolerances(file_name: str):
```

Comment:
I think all of these tolerances will need some more work. I've run three InSAR comparisons now with these names:

```python

```
that could be used to determine the best tolerances. Would be best to compare:
* prod 1 vs prod 2
* prod 2 vs prod 3
* prod 3 vs prod 1
* test 1 vs test 2
* test 2 vs test 3
* test 3 vs test 1

to determine the tolerances

Reply:
The last run was successful with these tolerances: 

https://github.com/ASFHyP3/hyp3-testing/actions/runs/418569295

Reply:
Okay, I've got all the absolute tolerances written down for these comparisons -- ideally we'd figure out good relative tolerances so we can bring down the absolute tolerances. 

Comparisons calls: 

```
pytest -s --name 'hyp3-testing-9Apv1Hj' --golden-dirs '_test/hyp3-testing-9Apv1Hj/main' '_test/hyp3-testing-9Apv1Hj/develop' tests/test_tifs.py
pytest -s --name 'hyp3-testing-evcd52X' --golden-dirs '_test/hyp3-testing-evcd52X/main' '_test/hyp3-testing-pIcNMBf/develop' tests/test_tifs.py
pytest -s --name 'hyp3-testing-pIcNMBf' --golden-dirs '_test/hyp3-testing-pIcNMBf/main' '_test/hyp3-testing-pIcNMBf/develop' tests/test_tifs.py

pytest -s --name 'foo' --golden-dirs '_test/hyp3-testing-9Apv1Hj/main' '_test/hyp3-testing-evcd52X/main' tests/test_tifs.py
pytest -s --name 'foo' --golden-dirs '_test/hyp3-testing-evcd52X/main' '_test/hyp3-testing-pIcNMBf/main' tests/test_tifs.py
pytest -s --name 'foo' --golden-dirs '_test/hyp3-testing-pIcNMBf/main' '_test/hyp3-testing-9Apv1Hj/main' tests/test_tifs.py

pytest -s --name 'foo' --golden-dirs '_test/hyp3-testing-9Apv1Hj/develop' '_test/hyp3-testing-evcd52X/develop' tests/test_tifs.py
pytest -s --name 'foo' --golden-dirs '_test/hyp3-testing-evcd52X/develop' '_test/hyp3-testing-pIcNMBf/develop' tests/test_tifs.py
pytest -s --name 'foo' --golden-dirs '_test/hyp3-testing-pIcNMBf/develop' '_test/hyp3-testing-9Apv1Hj/develop' tests/test_tifs.py

```